<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderStock</class>
 <widget class="QDialog" name="OrderStock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1265</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView_incoming_orders">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>351</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_products">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>361</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_suplier">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>331</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_suplier">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>104</width>
     <height>41</height>
    </rect>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select_suplier">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Select suplier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select_product">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>230</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Select product</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_product">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>104</width>
     <height>41</height>
    </rect>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_amount">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>104</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Suplier</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Product</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add_product">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>530</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add product to order</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_finish">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>530</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish order</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_create_order">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>421</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
